docs/docs/generating-nlu-data.mdx

```diff
@@ -52,6 +52,39 @@ Your assistant will always make mistakes initially, but
 the process of training & evaluating on user data will set your model up to generalize
 much more effectively in real-world scenarios.
 
+### Data Augmentation
```

Comment:
Should I clarify that data augmentation is different from a fixed script because it can create syntactic and semantic variation, but obv is still second to real user data?

Reply:
Yeah I mean we just need to make sure we're not contradicting ourselves in one section to the next 😅